ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml

```diff
@@ -353,3 +353,102 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )v4.3-integration-e2e-benchmark,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: api.ci
+    context: ci/prow/v4.4-integration-e2e-benchmark
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      ci-operator.openshift.io/variant: v4.4
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-odo-master-v4.4-integration-e2e-benchmark
+    rerun_command: /test v4.4-integration-e2e-benchmark
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --kubeconfig=/etc/apici/kubeconfig
+        - --lease-server-password-file=/etc/boskos/password
+        - --lease-server-username=ci
+        - --lease-server=https://boskos-ci.svc.ci.openshift.org
+        - --secret-dir=/tmp/secret
```

Comment:
@amitkrout Are we going to keep it as a future clearance of issue : https://github.com/openshift/odo/issues/2537?

Reply:
This is created by the job generator. Later i will clean up those. BTW keeping those configuration is not hurting us atm.  

Anyway good catch. I will create a separate issue/pr to cleanup those from all cluster configuration.